Makefile

```diff
@@ -79,9 +78,34 @@ $(POLKADOT_RUNTIME_WASM):
 	git submodule update --init --recursive -- $(POLKADOT_SUBMODULE)
 	cd $(POLKADOT_SUBMODULE) && cargo build --package node-template --release
 
+# Specification Build
+# -------------------
+
+SPEC_NAMES := set-free-balance
+
+SPECS_DIR := $(BUILD_DIR)/specs
+ALL_SPECS := $(SPECS_DIR)/$(SPEC_NAMES:=-spec.k)
+
+$(SPECS_DIR)/%-spec.k: %.md
+	mkdir -p $(SPECS_DIR)
```

Comment:
Should we escape this somehow, or are these relative paths?

Reply:
I'm not sure I follow. These are relative paths, `SPECS_DIR == .build/specs`.